.github/workflows/vpnmanager-deploy-dev.yml

```diff
@@ -57,6 +57,7 @@ jobs:
             SENTRY_ENVIRONMENT=${{ env.SENTRY_ENVIRONMENT }}
             SENTRY_ORG=${{ secrets.SENTRY_ORG }}
             SENTRY_PROJECT=${{ secrets.VPNMANAGER_SENTRY_PROJECT }}
+            SECRET_TOKEN=${{ secrets.VPNMANAGER_SECRET_TOKEN }}
```

Comment:
Secret for what? e.g. VPNMANAGER_SECRET_TOKEN, VPNMANAGER_API_SECRET/KEY, etc.

Reply:
API secret key to ensure the script is not abused.